frontend/src/components/core/Container.tsx

```diff
@@ -12,24 +12,28 @@ const createContainer = (Container: any, props: MyProps) => {
 
 export const FlexContainer = styled.div`
   display: flex;
-  flex-wrap: wrap;
-  max-height: 100vh;
+  flex: auto;
+  flex-direction: column;
   overflow: hidden;
+  position: relative;
```

Comment:
Lines 15 and 16 here ensure the splitter box and the room info bar don't overlap; the room info bar also seems to lose its centered position when they are removed. As for `position: relative`, the splitter box will extend beyond the page length otherwise. Without `flex:auto` the `SplitterContainer` will have a height of 0. I'm not sure why it's required in both places.

Reply:
Awesome, sounds good -- thanks!